plugins/frontend/backstage-plugin-prometheus/src/hooks/usePrometheus.ts

```diff
@@ -132,10 +132,23 @@ export function useAlerts(alerts: string[] | 'all') {
     }, []);
   if (value && value.status !== 'error') {
     const rules = value.data.groups.flatMap(it => it.rules);
+    const labels = getLabels(entity);
+    const parsedLabels = labels
+      ? Object.fromEntries(labels.split(',').map(label => label.split('=')))
+      : undefined;
     const displayableAlerts: PrometheusDisplayableAlert[] = rules
       .filter(rule => alerts === 'all' || alerts.includes(rule.name))
       .flatMap(rule =>
         rule.alerts.map(alert => ({ ...rule, ...alert, id: rule.name })),
+      )
+      .filter(alert =>
+        parsedLabels
+          ? Object.keys(parsedLabels).every(
```

Comment:
Can this ternary be replaced with something better? Like `&&` or `||`? Returning just `true` on the else branch screams that somethings wrong and ternary is not really necessary...